config/gradle/common.gradle

```diff
@@ -27,9 +27,16 @@ tasks.withType(JavaCompile) {
 
 // We use both Maven Central and our own Artifactory instance, which contains module builds, extra libs, and so on
 repositories {
-    // External libs - jcenter is Bintray and is supposed to be a superset of Maven Central, but do both just in case
-    jcenter()
-    mavenCentral()
+    // External libs - jcenter is Bintray and a superset of Maven Central
+    jcenter {
+        content {
+            // This is first choice for most java dependencies, but assume we'll need to check our
+            // own repository for things from our own organization.
+            // (This is an optimization so gradle doesn't try to find our hundreds of modules
+            // in jcenter.)
+            excludeGroupByRegex(/org\.terasology(\..+)?/)
+        }
```

Comment:
This change looks reasonable to me, and the comment explains well what is happening here.